tests/test_rule_in_container.sh

```diff
@@ -185,7 +185,7 @@ script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)" || { echo "Couldn't d
 # [ <-- needed because of Argbash
 
 podman images | grep -q "$_arg_name" || die "Couldn't find the podman image '$_arg_name'"
-test_image_cpe_product=$(podman run --rm "$_arg_name" cat /etc/os-release | grep cpe | cut -d : -f 4)
+test_image_cpe_product=$(podman run --rm "$_arg_name" cat /etc/os-release | grep CPE_NAME | cut -f 2 -d = | tr -d "'\"")
```

Comment:
What about something like:

```
bash -c 'source /etc/os-release && echo $CPE_NAME'
```

`/etc/os-release` & friends were IIRC meant [to be source-able](https://www.freedesktop.org/software/systemd/man/os-release.html) and this is shorter. :-)

Reply:
So that's not so simple as source is a bash builtin, but something very similar works.